image.yaml

```diff
@@ -142,13 +102,12 @@ envs:
       description: Disable the boot errors check in the probes.
 ports:
     - value: 8443
-    - value: 8778
 modules:
       repositories:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-14
+                  ref: master
```

Comment:
I suppose we'll need to set this to Sprint 16 after merging?

Reply:
Yes, it's safer option for the case there are sprint-17 WIP cct_modules changes merged already, that could cause the build not to be stable.

Reply:
I doens't seem to be able to edit the content of this auto-generated GH PR. If that ```cct_module``` reference is the only issue, could we fix it post merging -- I will do a PR for that.

Reply:
Yes.  I recommend moving to an overrides-release.yaml file.  When you do your release builds, just specify --overrides=./overrides-release.yaml.  You can then leave `ref: master` in the image.yaml file.

overrides-release.yaml:
```yaml
modules:
  repositories:
  - name: cct_module
    git:
      url: https://github.com/jboss-openshift/cct_module.git
      ref: sprint-14
```

This will alleviate merge issues in the future (since -dev is always using `master` and the release branches are always using `sprint-x`).